back-end/aws-cloud9-api/src/main/java/com/serverless/dal/Form.java

```diff
@@ -1,128 +1,103 @@
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBAutoGeneratedKey;
 import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBHashKey;
 import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBRangeKey;
-import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBTable;
-import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBQueryExpression;
-import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBScanExpression;
-import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBAutoGeneratedKey;
-import com.amazonaws.services.dynamodbv2.datamodeling.PaginatedQueryList;
-import com.amazonaws.services.dynamodbv2.model.AttributeValue;
-import com.serverless.Handler;
-
-import java.io.IOException;
+import java.util.Collections;
+import java.util.Map;
 import java.util.HashMap;
 import java.util.List;
+import java.util.ArrayList;
 
-import org.apache.logging.log4j.LogManager;
-import org.apache.logging.log4j.Logger;
-import com.serverless.dal.DynamoDBAdapter;
-@DynamoDBTable(tableName = "PLACEHOLDER_PRODUCTS_TABLE_NAME")
 public class Form {
-
-    // get the table name from env. var. set in serverless.yml
-    private static final String FORMS_TABLE_NAME = System.getenv("FORMS_TABLE_NAME");
-
-    private static DynamoDBAdapter db_adapter;
-    private final AmazonDynamoDB client;
-    private final DynamoDBMapper mapper;
-
-    private Logger logger = LogManager.getLogger(this.getClass());
-    private String id;
-    private String name;
-    private String description;
-
-    @DynamoDBHashKey(attributeName = "id")
-    @DynamoDBAutoGeneratedKey
-    public String getId() {
-        return this.id;
-    }
-    public void setId(String id) {
-        this.id = id;
-    }
-
-    @DynamoDBRangeKey(attributeName = "name")
-    public String getName() {
-        return this.name;
-    }
-    public void setName(String name) {
-        this.name = name;
-    }
-
-    @DynamoDBAttribute(attributeName = "description")
-    public String getDescription() {
-        return this.description;
-    }
-    public void setDescription(String description) {
-        this.description = description;
-    }
-
-    public Form() {
-        // build the mapper config
-        DynamoDBMapperConfig mapperConfig = DynamoDBMapperConfig.builder()
-            .withTableNameOverride(new DynamoDBMapperConfig.TableNameOverride(FORMS_TABLE_NAME))
-            .build();
-        // get the db adapter
-        this.db_adapter = DynamoDBAdapter.getInstance();
-        this.client = this.db_adapter.getDbClient();
-        // create the mapper with config
-        this.mapper = this.db_adapter.createDbMapper(mapperConfig);
-    }
-
-  
-    @Override
-	public String toString() {
-		return "Form [getId()=" + getId() + ", getName()=" + getName() + ", getDescription()=" + getDescription() + "]";
+	
+	 private String id;
+	 private String name;
+	 private String creationDate;
+	 private List<Map<String,String>> users;
```

Comment:
Na pewno nie potrzebujemy jednej struktury danych, która przechowuje inne struktury danych. Wydaje mi się, że wystarczy nam zwykła lista z id użytkowników. A nawet jeśli będzie nam potrzebne id rekrutera, to wtedy wystarczy sama mapa (mapa działa jak lista, tylko zamiast indeksów ma klucze, którymi wtedy byłyby userId).
Wydaje mi się, że chciałeś po prostu parować userID i recrutairId, jeśli koniecznie potrzebowałbyś listy zamiast mapy, to trzeba by było to zrobić tak:
`private List<Pair<String,String>> users;`
Typ `Pair` nie jest kontenerem i przechowuje tylko jedną parę wartości

Reply:
Na razie zostawiłem jednak te mapy, ponieważ w DynamoDB jest typ "Map" ale nie ma typu "Pair"